frontend/src/config.ts

```diff
@@ -130,6 +134,10 @@ export function isEmbedded() {
     return config.jwt != null;
 }
 
+export function isServerless() {
+    return config.isServerless;
+}
```

Comment:
[question] Is it possible that the config is being accessed prematurely by calling the `isServerless` function, causing MobX to throw an error?